models/.gitattributes

```diff
@@ -7,7 +7,7 @@
 abp-models/* filter=lfs diff=lfs merge=lfs -text
 datasets/training-data/** filter=lfs diff=lfs merge=lfs -text
 datasets/validation-data/** filter=lfs diff=lfs merge=lfs -text
-dlp_models/gliner_bi_encoder/** filter=lfs diff=lfs merge=lfs -text
+dlp_models/gliner_bi_encoder/*.json filter=lfs diff=lfs merge=lfs -text
```

Comment:
Not a blocker, but I noticed that the naming convention of the models/ directory is using `-` (so this folder would be `dlp-models` instead.